libs/designsystem/src/lib/components/modal/services/modal.controller.ts

```diff
@@ -22,6 +22,9 @@ export class ModalController implements IModalController {
   ) {}
 
   public showModal(config: ModalConfig, onCloseModal?: (data?: any) => void): void {
+    if (config.dim != undefined) {
```

Comment:
Consider using `!==` instead of `!=`?

Not sure if it would make sense to do:
```suggestion
    if (!config.dim) {
```
instead, perhaps we want to check for not using the property key `dim` at all here (which would make the current solution perfect).

Reply:
We ARE actually checking for whether the property is provided or not - so the most accurate would be: `config.hasOwnProperty('dim')`

Reply:
Fixed